### PR TITLE
2022-Verhagen-001 first version

### PR DESCRIPTION
**Pull request by LimesLimits**

### CODE REVIEW

File: 2022-Verhagen-001/NASSA.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+id: 2022-Verhagen-001
+nassaVersion: 0.3.0
+moduleType: Algorithm
+title: Fertility_Rate.nlogo
```
Comment (Andros-Spica) [inline 2022-Verhagen-001/NASSA.yml:4]: please make sure that the title is briefly explaining what the module does

File: 2022-Verhagen-001/NASSA.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+moduleVersion: 1.0.0
+contributors:
+ - name: Verhagen, Philip (with no accent marks, first upper-case letters, and surname matching the id and root directory name)
```
Comment (Andros-Spica) [inline 2022-Verhagen-001/NASSA.yml:7]: please delete text from the template

File: 2022-Verhagen-001/NASSA.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+   roles: [ "Author", "Compiler", "Contributor", "Copyright Holder", "Creator", "Thesis Advisor", "Translator" ]
```
Comment (Andros-Spica) [inline 2022-Verhagen-001/NASSA.yml:8]: please select only the relevant roles and delete the others

File: 2022-Verhagen-001/README.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# Module name or title
```
Comment (Andros-Spica) [inline 2022-Verhagen-001/README.md:1]: please place title from NASSA.yml

File: 2022-Verhagen-001/documentation/designDetails.md [hunk 5]
```diff
@@ -0,0 +1 @@
+# Design details
```
Comment (Andros-Spica) [inline 2022-Verhagen-001/documentation/designDetails.md:1]: delete any empty files

**Review by Andros-Spica — CHANGES_REQUESTED** [review]

File: 2022-Verhagen-001/NASSA.yml (hunk 1 above)
Comment (LimesLimits) [reply 2022-Verhagen-001/NASSA.yml:4]: done

File: 2022-Verhagen-001/NASSA.yml (hunk 2 above)
Comment (LimesLimits) [reply 2022-Verhagen-001/NASSA.yml:7]: done

File: 2022-Verhagen-001/NASSA.yml (hunk 3 above)
Comment (LimesLimits) [reply 2022-Verhagen-001/NASSA.yml:8]: done

File: 2022-Verhagen-001/README.md (hunk 4 above)
Comment (LimesLimits) [reply 2022-Verhagen-001/README.md:1]: done

File: 2022-Verhagen-001/documentation/designDetails.md (hunk 5 above)
Comment (LimesLimits) [reply 2022-Verhagen-001/documentation/designDetails.md:1]: done

**Pull request merged**